src/content/en/updates/2018/03/smooshgate.md

```diff
@@ -0,0 +1,217 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Recently the internet was on fire because of #SmooshGate. What is it about, and where do we go from here? This write-up gives an overview.
+
+{# wf_updated_on: 2018-03-19 #}
+{# wf_published_on: 2018-03-19 #}
+{# wf_tags: javascript #}
+{# wf_featured_image: /web/updates/images/generic/info.png #}
+{# wf_featured_snippet: Recently the internet was on fire because of #SmooshGate. What is it about, and where do we go from here? This write-up gives an overview. #}
+{# wf_blink_components: Blink>JavaScript>Language #}
+
+# #SmooshGate FAQ {: .page-title }
+
+{% include "web/_shared/contributors/mathiasbynens.html" %}
+
+## What the _smoosh_ happened?! {: #what }
+
+A [proposal](https://tc39.github.io/proposal-flatMap/) for a JavaScript
+language feature called `Array.prototype.flatten` turns out to be
+Web-incompatible. Shipping the feature in Firefox Nightly caused [at least
+one popular website](https://bugzilla.mozilla.org/show_bug.cgi?id=1443630)
+to break. Given that the problematic code is part of the (once extremely
+popular) MooTools library, it’s likely that many more websites are affected.
```

Comment:
this reads like it's no longer used, which I think is the point. It was popular and is still on production sites.

Reply:
I’ll clarify.